src/optimade_maker/logger.py

```diff
@@ -1,9 +1,4 @@
 import logging
 
-logging.basicConfig(
-    level=logging.DEBUG,
-    format="%(asctime)s - %(name)s - %(levelname)s - %(message)s",
-    datefmt="%Y-%m-%d %H:%M:%S",
-)
-
 LOGGER = logging.getLogger("optimake")
+LOGGER.setLevel(logging.INFO)
```

Comment:
Just FYI the previous config was putting pymongo into debug logging mode too, which is quite unpleasant :upside_down_face: